src/code/z_player_lib.c

```diff
@@ -570,7 +570,7 @@ s32 func_801235DC(PlayState* play, f32 arg1, s16 arg2) {
     return false;
 }
 
-ItemId func_8012364C(PlayState* play, Player* player, s32 arg2) {
+ItemId Player_GetItemOnButton(PlayState* play, Player* player, s32 arg2) {
```

Comment:
Looking at the usage of this function it seems like `arg2` is always passed a value equivalent to `EquipSlot`.
```suggestion
ItemId Player_GetItemOnButton(PlayState* play, Player* player, EquipSlot slot) {
```
thoughts?

Reply:
I was about to do this before PR'ing, but the one call to this function in z_player_lib.c made me hesitate, i.e. in function `func_80123810` since the arg `i` seems shifted up by 1 from C btns. Is `i` suppose to be `EquipSlot` when it's fed into this function?

Reply:
I wouldn't say it is shifting from an enum of just C buttons to another enum, but instead it is just accounting for the fact that `sCItemButtons` doesn't include the B button. That array could have comments like the following considering how it is used
```c
u16 sCItemButtons[] = {
    BTN_CLEFT, // EQUIP_SLOT_C_LEFT
    BTN_CDOWN, // EQUIP_SLOT_C_DOWN
    BTN_CRIGHT, // EQUIP_SLOT_C_RIGHT
};
```

Reply:
I would argue `sCItemButtons` is more of a map thats:
```c
typedef enum {
    /* 0 */ INTERACT_C_BTN_C_LEFT,
    /* 1 */ INTERACT_C_BTN_C_DOWN,
    /* 2 */ INTERACT_C_BTN_C_RIGHT,
    /* 3 */ INTERACT_C_BTN_MAX
} InteractCButton;
```
to controller buttons, then `i++` is really the macro `#define INTERACT_C_BTN_TO_BC_BTN(btnsC) ((btnsC) + 1)` being applied to `i` and storing the result back into `i`. So I'll leave the comments out for now until these different enums are introduced in MM.

For reference, equip slot would be `InteractBCAButton` from that PR, but then it might make more sense for `Player_GetItemOnButton` to take `InteractBCButton` as it's enum arg and not `InteractBCAButton`. Just something to be careful of